Vision/Vision.bs

```diff
@@ -107,76 +99,75 @@ Boilerplate: omit conformance
+	while continuing to expand the Web’s scope and reach. 
+	The Operational Principles for the W3C should reflect the core values of the Web itself.
+
+	These core values will be clearly demonstrated by how W3C leads the Web forward: 
+	by being inclusive, principled, and continually striving to make the Web better 
+	through these principles and the Ethical Web Principles. 
+	As the Ethical Web Principles state, “The web should empower an equitable, informed and interconnected society.”
+
+# Operational Principles for W3C # {#op-principles}
+
+	The fundamental function of W3C is to provide an open forum
 	where diverse voices from around the world
 	and from different organizations and industries
 	work together to build consensus
 	on voluntary global standards for Web technologies.
+	In order to fulfill this function, we will follow these operational principles:
 
-	* We put the needs of users first:
+	* W3C intentionally involves stakeholders from end to end 
+		in building the Web: developers, content creators, and end users. 
+		Our work will not be dominated by any person, company, or interest group, 
+		but uses consensus from the community to 
+		put the needs of users first: 
 		above authors, publishers, implementers, paying W3C Members, or theoretical purity.
-	* This organization believes in diversity
+	* We believe in diversity
 		and inclusion of participants from different
 		geographical locations,
 		cultures,
 		languages,
 		disabilities,
 		gender identities,
+		industries,
+		organizational sizes,
 		and more.
-	* We believe in principled, community-wide consensus-building
-		as the basis for building standards.
-	* We strongly emphasize accessibility,
+	* We create standards with an ethical intent to improve 
+		equity of access and participation on the Web. 
+		We will ensure the web platform meets broad goals 
+		using consistent horizontal review to ensure accessibility, 
 		internationalization,
+		sustainability,
 		privacy,
 		and security.
-	* We believe the Web must be an environmentally sustainable platform,
-		as defined in the [[ETHICAL-WEB-PRINCIPLES inline|Ethical Web Principles]].
 	* Our standards are rooted in a strong royalty-free patent policy
-		and open copyright licenses,
-		openly developed with consensus of industry and key stakeholders.
+		and open copyright licenses.
 	* We welcome individuals and organizations of all sizes
 		(from single-person companies to multi-nationals),
 		and take feedback from the general public.
-
-
-	# Principles and values # {#principles}
-
-	As W3C leads the Web forward,
-	our mission is to recognize and embody fundamental values and principles
-	into the architecture of the Web.
-	We must become more principled in our execution of the vision of the Web.
-	We must
-	* Ensure the Web is trustworthy,
-		by ensuring security and privacy for users.
-	* Aim to reduce centralization in Web architecture,
+	* We believe wide and interoperable implementation is a 
```

Comment:
Interop is critical. Note the following pulls in some themes from previous multiple lists.